influxql/_v1tests/server_helpers.go

```diff
@@ -0,0 +1,221 @@
+package v1tests
+
+import (
+	"context"
+	"encoding/json"
+	"io/ioutil"
+	"net/http"
+	"net/url"
+	"strings"
+	"testing"
+	"time"
+
+	"github.com/influxdata/influxdb/v2"
+	icontext "github.com/influxdata/influxdb/v2/context"
+	"github.com/influxdata/influxdb/v2/tests"
+	"github.com/influxdata/influxdb/v2/tests/pipeline"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+	"go.uber.org/zap/zapcore"
+	"go.uber.org/zap/zaptest"
+)
+
+func OpenServer(t *testing.T, extra ...tests.PipelineOption) *tests.DefaultPipeline {
+	t.Helper()
+
+	defaults := []tests.PipelineOption{
+		tests.WithLogger(zaptest.NewLogger(t, zaptest.Level(zapcore.ErrorLevel))),
+	}
+
+	p := tests.NewDefaultPipeline(t, append(defaults, extra...)...)
```

Comment:
Are there any config parameters (i.e. ports, data file paths) that we should be randomizing for each launcher to avoid problems with parallel tests?